docs/source/conf.py

```diff
@@ -22,10 +22,10 @@
 # -- Project information -----------------------------------------------------
 
 project = "lightly"
-copyright_year = "2020"
+copyright_year = "2020-<script>document.write((new Date()).getFullYear())</script>"
 copyright = "Lightly AG"
 website_url = "https://www.lightly.ai/"
-author = "Philipp Wirth, Igor Susmelj"
+author = "Philipp Wirth, Igor Susmelj, Guarin Flück, Malte Ebner, Jeremy A. Prescott"
```

Comment:
We can just say `Lightly AG`, that way everyone is inluded also in the future (like myself :) )

Reply:
upps :shame: didn't forget you on purpose. Will adjust it to Lightly AG :)